camus-etl-kafka/src/main/java/com/linkedin/camus/etl/kafka/common/KafkaReader.java

```diff
@@ -238,31 +239,42 @@ public boolean fetch() throws IOException {
                     Iterator<MessageAndOffset> messageIter2 = messageBuffer
                             .iterator();
                     MessageAndOffset message = null;
+                    int messageCount = 0;
                     while (messageIter2.hasNext()) {
+                    	messageCount++;
                         message = messageIter2.next();
                         if (message.offset() < currentOffset) {
                             //flag = true;
                             skipped++;
                         } else {
-                            log.debug("Skipped offsets till : "
-                                    + message.offset());
+                            log.debug("Skipped offsets till : " + message.offset());
                             break;
                         }
                     }
+                    log.info("Messages fetched: " + messageCount);
                     log.debug("Number of offsets to be skipped: " + skipped);
                     while(skipped !=0 )
                     {
                         MessageAndOffset skippedMessage = messageIter.next();
-                        log.debug("Skipping offset : " + skippedMessage.offset());
+                        log.info("Skipping offset : " + skippedMessage.offset());
```

Comment:
Is it really a warning?
We just got some messages that are earlier than we requested. We just skip them and no worries.